src/main/java/net/atos/zac/util/MediaTypes.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.util
+
+/**
+ * List of media types used by the ZAC backend.
+ *
+ * See [Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml) of IANA.
+ *
+ * Thanks goes out to [Youngbin Kim](https://gist.github.com/retheviper) for providing the original code
+ * which contains a much more complete list of media types:
+ * https://gist.github.com/retheviper/9b4e28f66b354d9f706e43d399100676
+ */
+object MediaTypes {
+    /** Hypertext Application Language (HAL) for JSON */
+    const val MEDIA_TYPE_HAL_JSON = "application/hal+json"
+
+    /** Standardized format (RFC 7807) for representing error responses in HTTP APIs */
+    const val MEDIA_TYPE_PROBLEM_JSON = "application/problem+json"
+
+    enum class Application(val extensions: Array<String>, val mediaType: String) {
+        /** Microsoft Excel */
+        MS_EXCEL(arrayOf(".xls"), "application/vnd.ms-excel"),
```

Comment:
Do the extensions require dots?

Reply:
To be honest we don't even use these extensions at all at the moment. I copied this code from https://gist.github.com/retheviper/9b4e28f66b354d9f706e43d399100676 because it looked good but currently we only use the media types. We could decide to remove those extensions in the end if we have no use for them..

Reply:
For now I'll leave them in..